sound/hybrid_font.h

```diff
@@ -120,7 +132,49 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (swing_player_) {
+        // avoid overlapping swings, based on value set in ProffieOSSwingOverlap.  Value is
+        // between 0 (no overlap) and 1.0 (full overlap)
+        if (swing_player_->pos() / swing_player_->length() >= config_.ProffieOSSwingOverlap) {
+          RefPtr<BufferedWavPlayer> overlap_swing = swing_player_;
+          swing_player_ = PlayPolyphonic(&swng);
```

Comment:
If you move this down a few lines (after the Free()) then there is no need for the overlap_swing variable.

Reply:
Won’t that introduce a delay in switching swing sounds?

Reply:
No, and I'm not sure why you would think it would.

Reply:
seems to function the same with the overlay_swing variable removed and no fade.

Reply:
after deeper testing across multiple fonts, not having the combination of overlay_player_ and fade out causes odd cut off for some swings.  Having this separate player and the fade and stop function makes all overlays sound as natural as possible.

Reply:
we can still have fade and stop without having a separate variable.

Reply:
I'll test again with the proper fade time set in SetSwingVolume but for whatever reason if I don't use a separate variable in my testing, I get an odd 'hiccup' on swing starts for some fonts.

Reply:
If the code as written above gives you a hiccup, but the other code doesn't, then there is a bug somewhere, and we should find and fix it...

Reply:
I think the issue arises because we're setting volume on the swing_player so if we don't hand off the overlain swing to a different variable, we get odd audio effects because both swings are getting volume set by SetSwingVolume.

Reply:
hiccup issue resolved with correct fade-time variable set in SetSwingVolume.  a value of 0.02 fixes hum hiccups and seems to eliminate crackles.  Resolved.

Reply:
fixed, hiccup and crackles seem resolved with setting a short fade_time in SetSwingVolume (20ms)